packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -100,18 +97,10 @@ const ChallengeTeamWizard: React.FunctionComponent<
   const [membershipStatus, setMembershipStatus] = useState<
     Record<string, TeamMembershipStatus>
   >({})
-  const [inviteMembersSuccess, setInviteMembersSuccess] =
-    useState<boolean>(false)
-  const [registerChallengeSuccess, setRegisterChallengeSuccess] =
-    useState<boolean>(false)
-  const [newTeam, setNewTeam] = useState<CreateTeamRequest>({
-    name: '',
-    description: '',
-    message: '',
-    invitees: '',
-  })
```

Comment:
This component no longer directly handles creating a new team, registering a new team for a challenge, or inviting users, so don't track these in state